ddtrace/tracer/sqlcomment.go

```diff
@@ -88,20 +91,45 @@ func (c *SQLCommentCarrier) Inject(spanCtx ddtrace.SpanContext) error {
 			}
 		}
 		if globalconfig.ServiceName() != "" {
-			tags[sqlCommentService] = globalconfig.ServiceName()
+			tags[sqlCommentParentService] = globalconfig.ServiceName()
 		}
 		if env != "" {
-			tags[sqlCommentEnv] = env
+			tags[sqlCommentParentEnv] = env
 		}
 		if version != "" {
-			tags[sqlCommentVersion] = version
+			tags[sqlCommentParentVersion] = version
 		}
 		tags[sqlCommentDBService] = c.DBServiceName
 	}
 	c.Query = commentQuery(c.Query, tags)
 	return nil
 }
 
+// encodeTraceParent encodes trace parent as per the w3c trace context spec (https://www.w3.org/TR/trace-context/#version).
+func encodeTraceParent(traceID uint64, spanID uint64, sampled int64) string {
```

Comment:
👍 this is plenty readable to my 👀

Reply:
Yeah, I think having it in a function makes it just fine.